Python/structmember.c

```diff
@@ -4,8 +4,24 @@
 #include "Python.h"
 #include "pycore_abstract.h"      // _PyNumber_Index()
 #include "pycore_long.h"          // _PyLong_IsNegative()
+#include "pycore_object.h"        // _Py_TryIncrefCompare(), FT_ATOMIC_*()
+#include "pycore_critical_section.h"
 
 
+static inline PyObject *
+member_get_object(const char *addr, const char *obj_addr, PyMemberDef *l)
+{
+    PyObject *v = FT_ATOMIC_LOAD_PTR(*(PyObject **) addr);
+    if (v == NULL) {
+        PyObject *obj = (PyObject *) obj_addr;
+        PyTypeObject *tp = Py_TYPE(obj);
+        PyErr_Format(PyExc_AttributeError,
+                     "'%.200s' object has no attribute '%s'",
+                     tp->tp_name, l->name);
```

Comment:
Use the new formatting syntax, since we're touching it.

```suggestion
                     "'%N' object has no attribute '%s'",
                     tp, l->name);
```

Reply:
Thank you for taking a look!